From: "Barak A. Pearlmutter" <barak+git@cs.nuim.ie>
Date: Sun, 1 Jan 2012 19:13:15 +0100
Subject: man dash hyphen slash de

silence lintian warning about hyphens vs dashes in German man page
---
 vobcopy.1.de | 84 ++++++++++++++++++++++++++++++------------------------------
 1 file changed, 42 insertions(+), 42 deletions(-)

diff --git a/vobcopy.1.de b/vobcopy.1.de
index d47c387..794406d 100644
--- a/vobcopy.1.de
+++ b/vobcopy.1.de
@@ -6,31 +6,31 @@
 .SH NAME
 vobcopy \- kopiert (rippt) Dateien von einer (Video-) DVD auf die Festplatte
 .SH SYNTAX
-.B vobcopy [-b
+.B vobcopy [\-b
 .I Größe[bkmg]
-.B ] [-e
+.B ] [\-e
 .I Größe[bkmg]
-.B ] [-f] [-F
+.B ] [\-f] [\-F
 .I Beschleunigungsfaktor
-.B ] [-h] [-i
+.B ] [\-h] [\-i
 .I Eingabeverzeichnis
-.B ] [-l] [-m] [-m][-n
+.B ] [\-l] [\-m] [\-m][\-n
 .I Titel-Nummer
-.B ] [-o
+.B ] [\-o
 .I Ausgabeverzeichnis
-.B ] [-q] [-O
+.B ] [\-q] [\-O
 .I einzelne zu kopierende Datei 
-.B ] [-t
+.B ] [\-t
 .I Name
-.B ] [-v [-v]] [-x] [-I] [-V] [-L
+.B ] [\-v [\-v]] [\-x] [\-I] [\-V] [\-L
 .I Logdatei-Pfad
-.B ] [-1
+.B ] [\-1
 .I zusätzliches Ausgabeverzeichnis 1
-.B ] [-2
+.B ] [\-2
 .I zusätzliches Ausgabeverzeichnis 2
-.B ] [-3
+.B ] [\-3
 .I zusätzliches Ausgabeverzeichnis 3
-.B ] [-4
+.B ] [\-4
 .I zusätzliches Ausgabeverzeichnis 4
 .B ]
 .SH Beschreibung
@@ -53,76 +53,76 @@ Routinen ist als Erweiterung zu libdvdread (siehe deren Webseite) erh
 Beim Aufruf ohne jegliche Optionen wird der Titel mit den meisten Kapiteln in Dateien 
 von 2GB Größe in das aktuelle Verzeichnis kopiert.
 .SH OPTIONEN
-.IP "-b, --begin GRÖSSE[bkmg]"
+.IP "\-b, \-\-begin GRÖSSE[bkmg]"
 beginnt den Kopiervorgang erst ab der angegebenen Adresse. Die Größe 
 der Adresse kann mit den Abkürzungen b für 512-Byte-Blöcke, k für Kilobytes, 
 m für Mega- und g für Gigabytes angegeben werden. 
-Beispiel: vobcopy -b 500m beginnt bei 500MB nach dem Anfang der Datei 
+Beispiel: vobcopy \-b 500m beginnt bei 500MB nach dem Anfang der Datei
 bis zum Ende zu kopieren.
-.IP "-e, --end GRÖSSE[bkmg]"
-ähnlich wie -b gibt diese Option eine Größe vor dem Ende der Datei an, 
+.IP "\-e, \-\-end GRÖSSE[bkmg]"
+ähnlich wie \-b gibt diese Option eine Größe vor dem Ende der Datei an,
 an der der Kopiervorgang stoppt.
-.IP "-f, --force"
+.IP "\-f, \-\-force"
 erzwingt die Ausgabe in das angegebene Verzeichnis, auch wenn vobcopy denkt, 
 dass dort ist nicht genügend freier Platz vorhanden ist.
-.IP "-F, --fast Beschleunigungsfaktor"
+.IP "\-F, \-\-fast Beschleunigungsfaktor"
 beschleunigt das Kopieren um einen Faktor (experimentell).
 Der Beschleunigungsfaktor kann 1 bis 64 betragen
-.IP "-h, --help"
+.IP "\-h, \-\-help"
 Gibt die verfügbaren Optionen aus
-.IP "-i, --input-dir EINGABE-VERZEICHNIS"
+.IP "\-i, \-\-input\-dir EINGABE-VERZEICHNIS"
 gibt das Eingabeverzeichnis an (das Verzeichnis, auf das die DVD
 gemountet ist).
-.IP "-l, --large-file"
+.IP "\-l, \-\-large\-file"
 schreibt alle Daten in eine große Datei ( > 2GB, benötigt Unterstützung für große Dateien (LFS))
-.IP "-M, --longest"
+.IP "\-M, \-\-longest"
 wählt den Titel mit der längsten Spielzeit. Bei manchen DVDs wird der Hauptfilm besser gefunden als mit der Methode der meisten Kapitel, bei manchen schlechter.
-.IP "-m, --mirror"
+.IP "\-m, \-\-mirror"
 Spiegelt die gesamte DVD auf die Festplatte. Es wird ein Unterverzeichnis mit dem 
 DVD-Namen erzeugt und die gesamte Dateistruktur des Videoteils wird darunter 
 kopiert. 
-.IP "-n, --title-number TITEL-NUMMER"
+.IP "\-n, \-\-title\-number TITEL-NUMMER"
 gibt an, welcher Titel kopiert werden soll. Auf der 
 DVD heißt der erste Titel vts_01_x.vob (meist der Hauptfilm).
-.IP "-o, --output-dir AUSGABE-VERZEICHNIS"
-gibt das Ausgabeverzeichnis für die Kopien an. Der Wert "stdout" oder "-" 
+.IP "\-o, \-\-output\-dir AUSGABE-VERZEICHNIS"
+gibt das Ausgabeverzeichnis für die Kopien an. Der Wert "stdout" oder "\-"
 gibt auf der Standardausgabe aus. Nützlich für die Ausgabe nach /dev/null ;-) 
 Tipp: Wenn du vergisst umzuleiten, wird dein Terminal überflutet werden, 
 in diesem Fall tippe "reset" und Enter, um das Terminal zu retten.
-.IP "-q, --quiet"
+.IP "\-q, \-\-quiet"
 alle Informations- und Fehlerausschriften von vobcopy werden in die Datei 
 vobcopy.bla im aktuellen Verzeichnis anstatt in die Standardfehlerausgabe geschrieben
-.IP "-O, --onefile single_file(s)_to_rip"
+.IP "\-O, \-\-onefile single_file(s)_to_rip"
 gibt (eine) einzelne Datei(en) für das Kopieren an. Teile von Namen können 
 angegeben werden und alle Dateien, die die Namensteile enthalten, werden 
 kopiert. Dateinamen können als mit Komma getrennte Liste angegeben werden.
-Beispiel: -O video_ts.vob,bup kopiert die Datei video_ts.vob und alle Dateien,
+Beispiel: \-O video_ts.vob,bup kopiert die Datei video_ts.vob und alle Dateien,
 deren Namen bup enthält.
-.IP "-t, --name NAME"
+.IP "\-t, \-\-name NAME"
 Damit kann man der Ausgabedatei einen eigenen Namen geben, wenn einem der von 
-der DVD nicht gefällt. -t hallo erzeugt also ein hallo.vob. (stdout oder "-" sind nun 
+der DVD nicht gefällt. \-t hallo erzeugt also ein hallo.vob. (stdout oder "\-" sind nun
 nicht mehr unterstützt) Wenn du Namen mit Leerzeichen wie "Huh ich mag den Film" 
 angeben willst, schließe den Namen in Anführungszeichen ein.
-.IP "-v, --verbose"
+.IP "\-v, \-\-verbose"
 schreibt noch mehr Informationen aus.
-.IP "-v -v"
+.IP "\-v \-v"
 schreibt die Ausschriften in eine Logdatei im aktuellen Verzeichis um einen Fehlerreport 
 zu erzeugen.
-.IP "-x"
+.IP "\-x"
 überschreibt alle existierenden Dateien ohne weitere Rückfragen.
-.IP "-L LOGDATEI-PFAD"
+.IP "\-L LOGDATEI-PFAD"
 weist vobcopy an die Logdatei statt in den Standardpfad in den LOGDATEI-PFAD zu legen.
-.IP "-I, --info"
+.IP "\-I, \-\-info"
 gibt Informationen über die Titel, Kapitel und Teile der DVD aus.
-.IP "-V, --version"
+.IP "\-V, \-\-version"
 gibt die Versionsnummer von vobcopy aus.
-.IP "-1, --1st_alt_output_dir AUXILIARY-OUTPUT-DIR1"
+.IP "\-1, \-\-1st_alt_output_dir AUXILIARY-OUTPUT-DIR1"
 Wenn die DVD einfach nicht in das erste Ausgabeverzeichnis passen will 
-(mit -o angegeben), geht die Ausgabe in das hier angegebene Verzeichnis 
+(mit \-o angegeben), geht die Ausgabe in das hier angegebene Verzeichnis
 (und die folgenden 2ten, 3ten und 4ten) weiter -> die Dateien werden geteilt, 
 je nachdem wie viel Platz in jedem der Verzeichnisse noch blieb.
-Gib den Pfad _direkt_ hinter "-1" an, _kein_ Leerzeichen dazwischen lassen, 
-wenn Probleme auftreten, möglicherweise ist das auch bei -o nötig...)
+Gib den Pfad _direkt_ hinter "\-1" an, _kein_ Leerzeichen dazwischen lassen,
+wenn Probleme auftreten, möglicherweise ist das auch bei \-o nötig...)
 .SH FEHLER
 Vobcopy ist weiterhin unter Entwicklung, also können noch einige Neuerungen 
 erwarten werden.
